src/Code.js

```diff
@@ -140,6 +150,11 @@ function postToBlockchain() {
   }
 }
 
+function showVersion() {
+  SpreadsheetApp.getUi()
+    .alert(`MultiBaas for Google Sheets Version ${VERSION}`);
+}
+
```

Comment:
On further thought, let's show the deployment ID here as well. It could be very useful to help keep track of which sheet is using which deployment.